lib/utils.js

```diff
@@ -421,7 +421,7 @@ exports.sliceArgs = sliceArgs;
 
 function now(dialect) {
   const now = new Date();
-  if (['postgres', 'sqlite'].indexOf(dialect) === -1) {
+  if (['mysql','postgres', 'sqlite'].indexOf(dialect) === -1) {
```

Comment:
I guess the extra part is just ignored for regular dates in mysql? 

Does this trigger a warning?